src/services/thermalOvercrowdingService.ts

```
@@ -79,11 +119,165 @@ export class ThermalOvercrowdingService {
+
+    // Sensory Overload Formula:
+    // Factor 1: Temp elevation (0-40 pts) -> 4 pts per °F above baseline
+    const tempComponent = Math.min(40, tempDelta * 4.0);
+    // Factor 2: Occupancy pressure (0-35 pts) -> exponential crowding penalty
+    const occupancyComponent = Math.min(35, Math.pow(occupancyRatio, 1.8) * 35.0);
+    // Factor 3: Heat rise velocity (0-25 pts) -> rate of temperature spike per 10 mins
+    const velocityComponent = Math.min(25, heatRiseRatePer10Min * 6.25);
+
+    const sensoryOverloadIndex = Math.min(
+      100,
+      Math.round((tempComponent + occupancyComponent + velocityComponent) * 10) / 10
```

ESLint

[error] Insert `,`

(prettier/prettier)